### ISSUE TITLE
Uninstall Completely

### ISSUE THREAD

**Issue by 3e9b** [Palakis/obs-ndi#440]
How do I uninstall completely on MacOS?

**Comment by Palakis** [comment]
@3e9b Delete the `/Library/Application Support/obs-studio/plugins/obs-ndi` folder

**Comment by itaysk** [comment]
@Palakis  I do not have `plugins` under `obs-studio`. (the NDI source does show up in OBS). 
How should I uninstall?

```bash
~$ ls ~/Library/Application\ Support/obs-studio/
basic		global.ini	logs		plugin_config	profiler_data
```

**Comment by sunknudsen** [comment]
@Palakis Does this also uninstall ndi-runtime-4.5.1-macOS.pkg?

**Comment by leafac** [comment]
@sunknudsen I had the exact same question and I think I found the answer: `ndi-runtime-4.5.1-macOS.pkg` installs the following files: `/usr/local/lib/libndi.4.dylib` and `/usr/local/lib/libndi_licenses.txt`. If you wish to uninstall `ndi-runtime-4.5.1-macOS.pkg`, those are the files should remove.

I discovered this using this application I just found called [Suspicious Package](https://mothersruin.com/software/SuspiciousPackage/).
